app/domain/Errors.scala

```diff
@@ -46,15 +46,14 @@ object InvalidPasswordError extends Error(ErrorCode.INVALID_PASSWORD, "Invalid p
 object PasswordRequiredError extends Error(ErrorCode.PASSWORD_REQUIRED, "Password is required")
 object TeamMemberAlreadyExistsInApplication extends Error(ErrorCode.USER_ALREADY_EXISTS, Messages("team.member.error.emailAddress.already.exists.field"))
 
-object ErrorCode extends Enumeration {
+sealed trait ErrorCode extends EnumEntry
 
-  type ErrorCode = Value
+object ErrorCode extends PlayEnum[ErrorCode] {
+  val values = findValues
 
-  val LOCKED_ACCOUNT = Value("LOCKED_ACCOUNT")
-  val BAD_REQUEST = Value("BAD_REQUEST")
-  val INVALID_PASSWORD = Value("INVALID_PASSWORD")
-  val PASSWORD_REQUIRED = Value("PASSWORD_REQUIRED")
-  val USER_ALREADY_EXISTS = Value("USER_ALREADY_EXISTS")
-
-  implicit val format = EnumJson.enumFormat(ErrorCode)
+  final case object LOCKED_ACCOUNT        extends ErrorCode
+  final case object BAD_REQUEST           extends ErrorCode
+  final case object INVALID_PASSWORD      extends ErrorCode
+  final case object PASSWORD_REQUIRED     extends ErrorCode
+  final case object USER_ALREADY_EXISTS   extends ErrorCode
 }
```

Comment:
This should be in another file to be consistent

Reply:
:+1: